core/chains/evm/gas/fixed_price_estimator.go

```diff
@@ -55,16 +55,9 @@ func (f *fixedPriceEstimator) GetDynamicFee(_ context.Context, originalGasLimit
 	if gasTipCap == nil {
 		return d, 0, errors.New("cannot calculate dynamic fee: EthGasTipCapDefault was not set")
 	}
-	chainSpecificGasLimit = applyMultiplier(originalGasLimit, f.config.EvmGasLimitMultiplier())
 
-	var feeCap *assets.Wei
-	if f.config.EvmGasBumpThreshold() == 0 {
-		// Gas bumping is disabled, just use the max fee cap
-		feeCap = getMaxGasPrice(maxGasPriceWei, f.config)
-	} else {
```

Comment:
`getMaxGasPrice` is also used inside `capGasPrice` - maybe it's possible to reuse `capGasPrice` here? (renaming would might also help since it's just a various minimize comparisons)

Reply:
I realised that might not work because the logic for `capGasPrice` is to find the highest gas price that is less than the default

For dynamic fee, its to find the highest gas price that could be higher than the default. 

Therefore it looks like the logic are different and maybe we shouldnt combine them together?

Reply:
👍🏼 I'm good with keeping them separate! I was just curious if they could be combined :)